experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -424,17 +424,20 @@ func (r *ExpanderReconciler) evaluateAndSavePlan(ctx context.Context, logger log
 	}
 
 	// read context in cr.namespace
+	var contextBytes []byte
 	contextcr := unstructured.Unstructured{}
 	contextcr.SetGroupVersionKind(contextGVK)
 	contextNN := types.NamespacedName{Namespace: cr.GetNamespace(), Name: "context"}
 	if err := r.Get(ctx, contextNN, &contextcr); err != nil {
 		logger.Error(err, "unable to fetch Context CR", "context", contextNN)
```

Comment:
Can we ignore _only_ `apierrors.IsNotFound(err)`, and still return an error if it's a different error.  The reason it matters is that you don't want e.g. timeout talking to kube-apiserver to cause a different, temporary reconciliation result.

Reply:
Ah yes. That was the intent.